python/cudf/cudf/core/column/datetime.py

```diff
@@ -977,6 +977,28 @@ def tz_convert(self, tz: str | None):
             "Cannot convert tz-naive timestamps, use tz_localize to localize"
         )
 
+    def to_pandas(
+        self,
+        *,
+        nullable: bool = False,
+        arrow_type: bool = False,
+    ) -> pd.Index:
+        if arrow_type and nullable:
```

Comment:
Could we do something like

```python
if not (arrow_type and nullable):
    return pd.Index(self.data_array_view(mode="read").copy_to_host())
return super().to_pandas(nullable=nullable, arrow_type=arrow_type)
```

instead? I think that should avoid the extra CPU copy.